src/main/java/net/pms/formats/v2/AudioProperties.java

```diff
@@ -219,4 +219,15 @@ public static int getSampleFrequencyFromLibMediaInfo(String mediaInfoValue) {
 			return result;
 		}
 	}
+
+	@Override
+	public String toString() {
+		StringBuilder result = new StringBuilder();
+		result.append("Channel(s): ").append(getNumberOfChannels());
+		if (getAudioDelay() != 0) {
+			result.append(", Delay: ").append(getAudioDelay());
+		}
+
```

Comment:
@Sami32 The thing is that someone did a half-ass implementation of ```AudioProperties```. Honestly I don't quite understand the need to have them as their own class, but if you move them they should be *moved* not *duplicated*. Whoever did it was probably afraid to break something or concerned with backwards compatibility for plugins - in any case, the situation now is a mess - as some properties are found two places and we have no control over which version is used by code. My guess is that some code use one and other another, which is a "bug-waiting-to-happen". But, I don't want to clean up such issues, as it always ends up in a huge argument about not changing ancient code that some plugin might rely on. As such, we should log both so we can at least see when they are different.